recipe/build.sh

```diff
@@ -25,6 +28,13 @@ if [[ "$channel_targets" == *conda-forge* && "${build_platform}" == "${target_pl
   export PATH=$SRC_DIR/cf-compilers/bin:$PATH
 fi
 
+GCC_CONFIGURE_OPTIONS=()
+
+if [[ "$channel_targets" == *conda-forge* ]]; then
+  GCC_CONFIGURE_OPTIONS+=(--with-pkgversion="conda-forge ${gcc_version}-${PKG_BUILDNUM}")
```

Comment:
Can you change this to `conda-forge gcc ${gcc_version}-${PKG_BUILDNUM}` to make sure that `gcc` is in the version string to support some scripts looking for gcc in the version string?

Reply:
Sure.  I'll open another PR to do that later this afternoon.

Reply:
#96